src/main/java/seedu/foodrem/model/item/ItemBoughtDate.java

```diff
@@ -16,22 +16,34 @@ public class ItemBoughtDate {
     public static final String BOUGHT_DATE_PATTERN_REGEX = "dd-MM-uuuu";
     public static final DateTimeFormatter BOUGHT_DATE_FORMATTER = DateTimeFormatter
             .ofPattern(BOUGHT_DATE_PATTERN_REGEX);
+
+    public static final ItemBoughtDate NOT_SET_BOUGHT_DATE = new ItemBoughtDate(LocalDate.MIN);
```

Comment:
Can we make this private and use a static method like `ItemBoughtDate::invalid` to return the instance instead?

Reply:
Realised the only time we need the instance is to check if the current date is not set. Will create a method called. "isNotSet()" instead.

Reply:
Good idea, thanks!